cmd/syncreboot/main.go

```diff
@@ -30,15 +30,26 @@ package main
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"os"
 	"syscall"
 )
 
+// Current software version.
+const version = "v1.0.0"
+
 func main() {
+	showVersion := flag.Bool("version", false, "show version")
 	shutdownPtr := flag.Bool("s", false, "shutdown system")
 	flag.Parse()
 
+	if *showVersion {
+		fmt.Println(version)
+		os.Exit(0)
+	}
+
+	syscall.Sync()
```

Comment:
Should this go inside the if statement? so that if we're rebooting, Sync() doesn't get called twice? (Probably not an issue but unnecessary.)

Reply:
I removed the second call.